smlb/metrics.py

```diff
@@ -579,6 +579,99 @@ def _evaluate(self, true, pred):
         return np.mean(ContinuousRankedProbabilityScore()._evaluate(true, pred))
 
 
+class StandardConfidence(ScalarEvaluationMetric):
+    '''Fraction of the time that the magnitude of a residual was less than the uncertainty estimate.
```

Comment:
```suggestion
    """Fraction of the time that the magnitude of the residual is less than the predicted standard deviation.
```
Being more specific because "uncertainty" has several meanings